BuildTools/.swiftlint.yml

```diff
@@ -85,9 +97,6 @@ excluded:
   - ${SRCROOT}/scripts/genstrings.swift
   - ${SRCROOT}/danger/DangerTests.swift
   - ${SRCROOT}/*UITests
-  - ${SRCROOT}/Tests
-  - ${SRCROOT}/*Tests
-  - ${SRCROOT}/*/Tests
   - ${SRCROOT}/.rbenv
   - ${SRCROOT}/*/*/*/*/main.swift # App Store Connect SDK structure
-  - ${SRCROOT}/*/*/Tests # App Store Connect SDK structure
```

Comment:
I've removed this line since we do want to lint all tests. Not sure what "App Store Connect SDK structure" means or what it was meant to stop from getting linted, but at least in Mule I am not seeing any strange things after I removed it. @AvdLee you see a reason to keep this in, without it preventing from linting all our tests?